packages/widget/src/components/common/dropdown/dropdown.ts

```diff
@@ -148,10 +175,11 @@ export class Dropdown extends BaseComponent {
             </div>
           </div>
           <div
+            part="dropdownContent"
             class="dropdownContent ${this.isDropdownOpen ? 'show' : ''}"
             role="list"
           >
-            ${this._renderOptions()}
+            ${this._renderOptions()} ${this.actionOption}
```

Comment:
nit: a tiny change, but i think it would make this more readable 🤷 
```suggestion
            ${this._renderOptions()}
            ${this.actionOption}
```

Reply:
prettier doesn't allow to push it to the next line